packages/cli/test/models/Dependency.test.js

```diff
@@ -2,11 +2,16 @@
 
 require('../setup');
 
+import { expect } from 'chai';
+import { accounts } from '@openzeppelin/test-environment';
+
 import sinon from 'sinon';
 import npm from 'npm-programmatic';
 import Dependency from '../../src/models/dependency/Dependency';
 
-contract('Dependency', function([_, from]) {
+const [from] = accounts;
```

Comment:
I think we should move these inside the `describe` block, to keep a style similar to the one `truffle test` uses and make these easier to read. This is what we're doing [in the contracts repo](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2007#discussion_r349930794)